Dockerfile

```diff
@@ -0,0 +1,29 @@
+# gammapy docker container
+FROM continuumio/miniconda3:latest
+RUN apt-get update && apt-get install -y build-essential python3-pip
```

Comment:
For what do you need build-essential and pip when using conda?

Also, please use docker image best practices, here cleaning up after using apt: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
You're right, we can remove `build-essential`, but `pip` is needed to do a development install, as justified in comments below..

Reply:
But pip comes with conda, no  need to install pip into the system

Reply:
Yes, but I have an error when running `pip install -e.` in the docker that does not happen if I install pip with apt-get...

Reply:
Then you are installing gammapy into the system, not into the conda env